subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -1049,6 +1064,19 @@ config BT_CTLR_ALLOW_SAME_PEER_CONN
 	  WARNING: This option enables behavior that violates the Bluetooth
 	  specification.
 
+config BT_CTLR_CONN_INTERVAL_LOW_LATENCY
+	bool "Allow low latency connection intervals"
+	help
+	  Allow low latency connection intervals.
```

Comment:
How is this expected to be used with the Zephyr host?

IIRC we test for valid HCI values (e.g. values under 7.5 cannot be requested), and if the connection completed event contain values lower than what's allowed with HCI, then that's considered invalid as well.

Reply:
Ah, forgot about that one :)